### PR TITLE
Advisor Dashboard UI

### LINKED ISSUE #228: Show All Students for Advisor View

### PR DESCRIPTION
**Pull request by Boccca2014**
Replace CommentForumContainer with Advisor Panel when isAdvisor is true.
Current progress: set up done for most components, need to add more functionality to show valid values.

Closes #228 
Closes #198

### CODE REVIEW

**Comment by miaboloix** [comment]
ping me when this is ready for review :)

**Comment by miaboloix** [comment]
I made it so that "Course List for <selected_student_name>" is displayed when an advisor selects a student

**Comment by miaboloix** [comment]
Definitely need a back button & styling before we merge

**Comment by Boccca2014** [comment]
A list of what needs to be changed is in issue #228

File: static/js/redux/ui/advising_schedule.jsx [hunk 1]
```diff
@@ -125,18 +125,8 @@ AdvisingSchedule.propTypes = {
...
-      author_name: PropTypes.string,
-      content: PropTypes.string,
-      timestamp: PropTypes.date,
-    })),
-    semester_name: PropTypes.string,
-    semester_year: PropTypes.string,
-
-  }),
+  // TODO: fix proptype validation of selected_advisee
+  selected_advisee: PropTypes.PropTypes.objectOf(PropTypes.string),
```
Comment (miaboloix) [inline static/js/redux/ui/advising_schedule.jsx:129]: the creates an error on the console!

File: static/js/redux/ui/course_list_row.jsx [hunk 2]
```diff
@@ -165,6 +165,8 @@ CourseListRow.propTypes = {
   courseToClassmates: PropTypes.shape({ '*': SemesterlyPropTypes.classmates }).isRequired,
   fetchCourseInfo: PropTypes.func.isRequired,
   timetableName: PropTypes.string.isRequired,
+  // TODO: fix proptype validation of selected_advisee
+  selected_advisee: PropTypes.PropTypes.objectOf(PropTypes.string),
```
Comment (miaboloix) [inline static/js/redux/ui/course_list_row.jsx:169]: its not an object of strings!

Comment (Boccca2014) [reply static/js/redux/ui/course_list_row.jsx]: I know, that's why I put the TODO above it lol

**Review by miaboloix — APPROVED** [review]

**Review by rbiz4 — APPROVED** [review]

**Pull request merged**